### PR TITLE
Add API for MAF at DSP section

### PR DESCRIPTION
**Pull request by dekorablev**

### CODE REVIEW

File: pkg/device/device.go [hunk 1]
```diff
@@ -22,23 +22,23 @@ import (
 )
 
 const (
-	Nch = 64
```
Comment (kozhukalov) [inline pkg/device/device.go]: Please, do not touch lines irrelevant to the issue you are trying to solve. This is the strict requirement.

Comment (dekorablev) [reply pkg/device/device.go]: But I dont!

Comment (kozhukalov) [reply pkg/device/device.go]: How on Earth you don't if there are plenty of lines changed?

**Comment by kozhukalov** [comment]
test

**Review by kozhukalov — CHANGES_REQUESTED** [review]
Please, rebase on main

**Review by kozhukalov — APPROVED** [review]

**Pull request merged**